src/list.json

```diff
@@ -275,6 +275,14 @@
       }
     ],
     "U": [
+      {
```

Comment:
Please Re-Style using this command in the Repo Directory
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2341054/patch | git am`
`git push`